railgun/hack/deploy/build-single-manifests.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/../..
+STAGE_DIR=$(mktemp -d)
+
+mkdir $STAGE_DIR/rbac
+mkdir $STAGE_DIR/base
+
+cd $SCRIPT_ROOT
+
+# split RBAC roles from kubebuilder generated roles
+kustomize build ./config/rbac/ -o $STAGE_DIR/rbac
+cp $STAGE_DIR/rbac/rbac.authorization.k8s.io_v1_clusterrole_manager-role.yaml $STAGE_DIR/base/manager-role.yaml
+cp $STAGE_DIR/rbac/rbac.authorization.k8s.io_v1_clusterrolebinding_manager-rolebinding.yaml $STAGE_DIR/base/manager-rolebinding.yaml
+cp $STAGE_DIR/rbac/default_rbac.authorization.k8s.io_v1_role_leader-election-role.yaml $STAGE_DIR/base/leader-election-role.yaml
+cp $STAGE_DIR/rbac/default_rbac.authorization.k8s.io_v1_rolebinding_leader-election-rolebinding.yaml $STAGE_DIR/base/leader-election-rolebinding.yaml
+# TODO auth proxy service stuff?
```

Comment:
It looks like we need to resolve this TODO before merging.

Reply:
It was something I wasn't familiar with as much and wasn't immediately necessary (since it's only used for Prometheus). It also requires some environment-specific configuration (you must bind to your Prometheus user; we don't know who that is in advance) that we can't add to the generated manifests. https://github.com/Kong/kubernetes-ingress-controller/issues/1579